sigmastate/src/test/scala/sigmastate/utxo/examples/IcoExample.scala

```diff
@@ -553,4 +553,10 @@ class IcoExample extends SigmaTestingCommons
     println(ComplexityTableStat.complexityTableString)
   }
 
+  /** This is the last executed test suite, so this method is executed after all tests.
+    * We output statistics of how PrecompiledScriptProcessor cache was used. */
+  override protected def afterAll(): Unit = {
+    println(ErgoLikeTestInterpreter.DefaultProcessorInTests.cache.stats())
```

Comment:
Use logger instead of println

Reply:
See comment about logging in tests above.